languages/elixir/exercises/concept/conditionals/.docs/hints.md

```diff
@@ -0,0 +1,20 @@
+### General
+
+- The [atom type is described here][atom]
+
+### 1. Return the logging code label
+
+- You can use a [`cond/1` function][cond] to elegantly handle the various log codes.
+- You can use `===/2` to compare integers for strict type equality
```

Comment:
Is there a reference somewhere that you can point to instead of having the actual code here?